runtime/doc/api.txt

```diff
@@ -1580,12 +1580,14 @@ nvim_exec({src}, {output})                                       *nvim_exec()*
     On execution error: fails with VimL error, updates v:errmsg.
 
     Parameters: ~
-      • {src}     Vimscript code
-      • {output}  Capture and return all (non-error, non-shell |:!|) output
+      • {src}   Vimscript code
+      • {opts}  Optional parameters.
+                • output: (boolean, default false) Whether to capture and
+                  return all (non-error, non-shell |:!|) output.
```

Comment:
Should `output` param be a string, is it likely that a caller would want to also capture "error" output in the future?

Reply:
I am not sure here. To me having a one-to-one relation between `opts` and returned dictionary seems like a better design right now. So if `opts.output = true`, then `output` in result will represent output. If `opts.error = true`, then errors will be safely captured and result `error` will have those possible error captures. And so on.